src/main/java/de/retest/recheck/report/ActionReplayResult.java

```diff
@@ -88,84 +68,42 @@ protected ActionReplayResult() {
 		description = null;
 		goldenMasterPath = null;
 		targetcomponent = null;
-		error = null;
-		targetNotFound = null;
 		stateDifference = null;
 		windows = null;
-		executionType = null;
 	}
 
 	public static ActionReplayResult createActionReplayResult( final ActionReplayData data,
-			final ExceptionWrapper error, final TargetNotFoundException targetNotFound,
 			final StateDifference difference, final long actualDuration, final SutState actualState ) {
-		if ( error != null || targetNotFound != null ) {
-			return withError( data, ErrorHolder.of( error, targetNotFound ) );
-		}
+
 		if ( difference != null && !difference.getRootElementDifferences().isEmpty() ) {
 			return withDifference( data, WindowRetriever.empty(), DifferenceRetriever.of( difference ),
 					actualDuration );
 		}
 		return withoutDifference( data, WindowRetriever.of( actualState ), actualDuration );
 	}
 
-	public static ActionReplayResult withError( final ActionReplayData data, final ErrorHolder error ) {
-		return new ActionReplayResult( data, WindowRetriever.empty(), error, DifferenceRetriever.empty(), NO_DURATION,
-				null );
-	}
-
 	public static ActionReplayResult withDifference( final ActionReplayData data, final WindowRetriever windows,
 			final DifferenceRetriever difference, final long duration ) {
-		return new ActionReplayResult( data, windows, ErrorHolder.empty(), difference, duration, null );
+		return new ActionReplayResult( data, windows, difference, duration );
 	}
 
 	public static ActionReplayResult withoutDifference( final ActionReplayData data, final WindowRetriever windows,
 			final long duration ) {
 		return withDifference( data, windows, DifferenceRetriever.empty(), duration );
 	}
 
-	public static ActionReplayResult insertion( final ActionState actionState ) {
-		return new ActionReplayResult( ActionReplayData.of( actionState.getAction() ),
-				WindowRetriever.of( actionState ), ErrorHolder.empty(), DifferenceRetriever.empty(),
-				actionState.getDuration(), ExecutionType.INSERTION );
-	}
-
-	public static ActionReplayResult deletion( final ActionState actionState ) {
-		return new ActionReplayResult( ActionReplayData.of( actionState.getAction() ),
-				WindowRetriever.of( actionState ), ErrorHolder.empty(), DifferenceRetriever.empty(), NO_DURATION,
-				ExecutionType.DELETION );
-	}
-
-	protected ActionReplayResult( final ActionReplayData data, final WindowRetriever windows, final ErrorHolder error,
-			final DifferenceRetriever difference, final long duration, final ExecutionType executionType ) {
-		if ( windows.isNull() && difference.isNull() && !error.hasError() ) {
+	protected ActionReplayResult( final ActionReplayData data, final WindowRetriever windows,
+			final DifferenceRetriever difference, final long duration ) {
+		if ( windows.isNull() && difference.isNull() ) {
```

Comment:
Could you reorder this constructor to the other constructor (i.e. above the static functions) ?